src/watts/plugin_sas.py

```diff
@@ -93,8 +95,9 @@ class PluginSAS(TemplatePlugin):
 
     def  __init__(self, template_file: str, show_stdout: bool = False,
                   show_stderr: bool = False,
-                  extra_inputs: Optional[List[str]] = None):
-        super().__init__(template_file, extra_inputs)
+                  extra_inputs: Optional[List[str]] = None,
+                  extra_template_inputs: Optional[List[PathLike]] = None):
+        super().__init__(template_file, extra_inputs, extra_template_inputs)
 
```

Comment:
@zhieejhia93 and @paulromano - those changes to SAS plugin need also to be pushed to all plugin (PyARC, MOOSE) that will benefit from this `extra_template_inputs` option - should this be done at a "higher level" (`plugin` class) or pushed to the other plugins?

Reply:
These are already in the `TemplatePlugin` class. I think we just need to add them to the MOOSE and PyARC plugins.

Reply:
OK - putting a new issue to make `extra_template_inputs` available on other plugins.

Reply:
Sure, I can handle that. Thanks.